### ISSUE TITLE
Getting 200 in refresh-api from empty token

### ISSUE THREAD

**Issue by coderbdsust** [ali-bouali/spring-boot-3-jwt-security#38]

**Comment by coderbdsust** [comment]
<img width="1260" alt="Screenshot 2023-07-22 at 1 42 18 PM" src="https://github.com/ali-bouali/spring-boot-3-jwt-security/assets/7193568/29ac87e9-f746-4656-8aff-a0303e72182f">

**Comment by ali-bouali** [comment]
can you share more context?
I didn't totally get the issue and you how you got it

**Comment by coderbdsust** [comment]
I fixed the issue, now it's look like this.
<img width="619" alt="Screenshot 2023-07-25 at 9 42 14 AM" src="https://github.com/ali-bouali/spring-boot-3-jwt-security/assets/7193568/5f5ba93e-a6c2-48bc-82f5-04b7886ce14a">

**Comment by ahmadbayan** [comment]
Yes, I had the same issue. I now throw an AuthorizationException, catch it in the controller and set the HTTP response code to 401 for such use cases where the token is missing on the refresh API. Thanks @coderbdsust for sharing this.
